operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/reconciler/Constants.java

```diff
@@ -7,13 +7,17 @@ public final class Constants {
 
   public static final Set<String> DEFAULT_NAMESPACES =
       Collections.singleton(Constants.WATCH_ALL_NAMESPACES);
-  public static final Set<String> WATCH_CURRENT_NAMESPACE_SET =
+  public static final Set<String> WATCH_CURRENT_NAMESPACES =
```

Comment:
Not sure about this, in this simple plural form it says we can watch more current namespace. While the case is that this is only a helper to wrap the WATCH_CURRENT_NAMESPACE. That is why I just added the SET suffix.

Reply:
Then let's rename all the Set constants with the `_SET` suffix

Reply:
agree!